avro4s-core/src/main/scala/com/sksamuel/avro4s/RecordFormat.scala

```diff
@@ -14,11 +13,10 @@ trait RecordFormat[T] extends ToRecord[T] with FromRecord[T] with Serializable
   */
 object RecordFormat {
 
-  def apply[T: Encoder : Decoder : SchemaFor]: RecordFormat[T] = apply(AvroSchema[T])
```

Comment:
No need for a separate SchemaFor, as Encoder and Decoder both provide a SchemaFor